app/javascript/vue/repository_item_sidebar/RepositoryItemSidebar.vue

```diff
@@ -300,7 +300,8 @@
             <div id="divider" class="w-500 bg-sn-light-grey flex px-8 items-center self-stretch h-px mb-6"></div>
             <div id="bottom-button-wrapper" class="flex h-10 justify-end">
               <button type="button" class="btn btn-primary print-label-button"
-                :data-rows="JSON.stringify([repositoryRowId])">
+                :data-rows="JSON.stringify([repositoryRowId])"
+                :data-repository-id="repository?.id">
```

Comment:
why safe navigation? repository should be always present I assume?

Reply:
this part of the code is rendered before we get information about the repository from the backend. So in this case this part of code would fail, without safe navigation